gadgets/trace_exec/gadget.yaml

```diff
@@ -93,3 +97,7 @@ ebpfParams:
     key: uid
     defaultValue: ""
     description: Show only events generated by processes with this uid
+  paths:
+    key: paths
+    defaultValue: "false"
+    description: Add the path of the executable
```

Comment:
For the CI to pass the params need to be in lexicographical order - So you need to put the `paths` before `targ_uid`